maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -396,10 +397,38 @@ class AndroidDriver(
         }) ?: throw IllegalStateException("Input Response can't be null")
     }
 
-    override fun openLink(link: String) {
+    override fun openLink(link: String, autoVerify: Boolean) {
         dadb.shell("am start -a android.intent.action.VIEW -d \"$link\"")
     }
 
+    fun autoVerifyApp(appId: String?, autoVerifyApp: (text: String) -> Unit) {
+        if (appId != null) {
+            val apkFile = AndroidAppFiles.getApkFile(dadb, appId)
```

Comment:
Just to double check - does this work on real devices?

Reply:
Yes I checked on my real device and was able to get app name. Should we add a try-catch around it and not fail this if somehow this throws an exception? WDYT?